.vscode/settings.json

```diff
@@ -4,5 +4,7 @@
       "javascriptreact",
       { "language": "typescript", "autoFix": true },
       { "language": "typescriptreact", "autoFix": true }
-    ]
+    ],
+    "eslint.alwaysShowStatus": true,
```

Comment:
Where is the status displayed?

Reply:
The status is shown in VS Code status bar